tests/unit/data_tasks/sql_query_test.py

```diff
@@ -70,8 +70,7 @@ def assert_query_null_pre_execute(self, query):
 
     @fixture
     def connection(self, db_session):
-        with db_session.bind.connect() as connection:
-            yield connection
+        return db_session.bind
```

Comment:
`Connect.connect()` is no longer supported. Probably because it doesn't make much sense, as `bind` here already is a connection object?